python/pyarrow/tests/test_array.py

```diff
@@ -732,19 +732,19 @@ def test_struct_array_sort():
         {"a": 5, "b": "foo"},
     ]
 
-    sorted_arr = arr.sort("descending")
+    sorted_arr = arr.sort("ascending", fieldname="b")
```

Comment:
This should be revert to omit the `fieldname`, it was the acceeptance test for sorting struct arrays wiithout any specific field.